src/test/java/net/openhft/chronicle/queue/internal/reader/ChronicleReaderTest.java

```diff
@@ -92,7 +92,8 @@ public void before() {
         if (OS.isWindows())
             if (!(testName.getMethodName().equals("shouldThrowExceptionIfInputDirectoryDoesNotExist") ||
                     testName.getMethodName().equals("shouldBeAbleToReadFromReadOnlyFile") ||
-                    testName.getMethodName().equals("shouldPrintTimestampsToLocalTime")))
+                    testName.getMethodName().equals("shouldPrintTimestampsToLocalTime") ||
+                    testName.getMethodName().equals("matchLimitThenNamedTailer")))
```

Comment:
Do we still get this even though we explicitly set readOnly to false?

Reply:
This condition is negated

Reply:
We could just use `ignoreException` here, it will not fail if the warning isn't present